recipes/bob.io.matlab/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: bob.io.matlab
+  version: {{ version }}
+
+source:
+  fn: bob.io.matlab-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.io.matlab/bob.io.matlab-{{ version }}.zip
+  md5: 089155b2d864a6dca0f30491da0ed4fc
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
for example, `bob.io.base` gets packages and has the file `lib/python3.5/site-packages/bob/io/__pycache__` in it. Now when I want to package `bob.io.matlab`, this one also has the the folder `lib/python3.5/site-packages/bob/io/__pycache__` which needs to copied. `conda` will try to copy this folder again and exits with an error like this (https://github.com/conda/conda-build/issues/858).